packages/flutter/lib/src/rendering/custom_layout.dart

```diff
@@ -291,7 +291,7 @@ abstract class MultiChildLayoutDelegate {
   /// This should compare the fields of the current delegate and the given
   /// `oldDelegate` and return true if the fields are such that the layout would
   /// be different.
-  bool shouldRelayout(covariant MultiChildLayoutDelegate oldDelegate);
+  bool shouldRelayout(covariant MultiChildLayoutDelegate? oldDelegate);
```

Comment:
shouldRelayout's argument should be non-nullable

Reply:
Changed to non-nullable